test/tools/ClientCertificateCheck/dockerfile

```diff
@@ -0,0 +1,18 @@
+# Stage 1: Build the Application
```

Comment:
@iSazonov  If we don't want to try this as a docker container, then no. I can remove the docker parts.